apps/www/src/lib/registry/default/example/CarouselPlugin.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import Autoplay from 'embla-carousel-autoplay'
+import { ref } from 'vue'
+import { Carousel, CarouselContent, CarouselItem, CarouselNext, CarouselPrevious } from '@/lib/registry/default/ui/carousel'
+import { Card, CardContent } from '@/lib/registry/default/ui/card'
+
+const plugin = ref(Autoplay({
```

Comment:
Why `ref`, is their Autoplay plugin is reactive when we change the options?

Reply:
No. It's just to be ensure that the autoplay plugin initialize only once.

Reply:
Maybe I should change the ref to reactive or so